background/get-addon-settings.js

```diff
@@ -187,6 +187,10 @@ chrome.storage.sync.get(["addonSettings", "addonsEnabled"], ({ addonSettings = {
             delete scratchr2.linkColor;
           }
         }
+        if (addonId == "project-info") {
+          madeChangesToAddon = madeAnyChanges = true;
+          settings.show = "text";
```

Comment:
It should default to text and icon, and be optout (users can turn off icon) - therefore no need for migration code